app/helpers/automatic/daily_room_pool.py

```
@@ -0,0 +1,487 @@
+            await self.delete_room(room.room_url)
+            logger.info(f"Deleted room for session {session_id}")
+        except Exception as e:
+            logger.error(f"Failed to delete room for session {session_id}: {e}")
+
+        # Replenish the pool.
+        if (
+            self.available_rooms.qsize() < self.pool_size
+            and not self.is_creating_room
+            and self._current_total_rooms() < self.max_pool_size
+        ):
+            asyncio.create_task(self._create_background_room())
```

Ruff (0.13.1)

Store a reference to the return value of `asyncio.create_task`

(RUF006)